src/content/docs/en/guides/content-collections.mdx

```diff
@@ -245,9 +267,52 @@ defineCollection({
 })
 ```
 
+### Defining collection references
+
+Collection entries can also "reference" other related entries. A common example is a blog post that references reusable author profiles stored as JSON, or related post URLs stored in the same collection.
+
+You can define these relationships using the `reference()` utility from your collection schema. This accepts a collection name and validates the entry identifier(s) in your frontmatter or data file.
```

Comment:
```suggestion
### Defining collection references

Collection entries can also "reference" other related entries. 

With the `reference()` function from the Collections API, you can define a property in a collection schema as an entry from another collection. For example, you can require that every `space-shuttle` entry includes a `pilot` property which uses the `pilot` collection's own schema for type checking, autocomplete and validation.

A common example is a blog post that references reusable author profiles stored as JSON, or related post URLs stored in the same collection:
```
This is how I'm thinking of it in my head, so I've spewed an example that is maybe a little more user-oriented re: "defining your property with another entry!"

Just throwing some alt wording at you to see whether you think a slight tweak in nuance is helpful here.

Reply:
LOVE this example :clap